addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -565,6 +565,16 @@ $o-wslides-fs-side-width: 300px;
     box-shadow: 3px 3px 5px gray;
 }
 
+// Make all the placeholders of class form-control inside elements of class form-group to have the same color
+
+.form-group .form-control::-webkit-input-placeholder {
```

Comment:
A bit strange as it would impact all addons ... unless this change is considered standard (in which case this should not be done in slides), we should always namespace to ensure we don't break other addons styling.

Reply:
Should I add another class that like slides-placeholder and change it here?

Reply:
But perhaps it would also make sense to change this everywhere (in website), as the current default colour for placeholder text is really dark. And hiding the text on focus also makes sense

Reply:
In which case this should be discussed with website team. Either we do something for slides custom, either we consider it standard and it is more website duty to be sure it is correctly done and adds value. I fear we will have unwanted side effects on other apps with this kind of global change.

Reply:
Comment not solved :/ (didn't pay attention because it was marked as solved)

and if you do it for website, you should not do it in the e-learning module but in the website module (but I think it's better to do the change only for e-learning)

![screenshot_spect](https://user-images.githubusercontent.com/54316226/140706099-8777b6b3-2c92-4ba1-95f7-08625ce97d7b.png)

Reply:
Hi, what about 
```css
.o_wslides_body .form-control::-webkit-input-placeholder {
```
? (see latest commit)